contracts/bonding/BondingManager.sol

```diff
@@ -982,13 +990,14 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
         // Increase delegate's delegated amount
         delegators[del.delegateAddress].delegatedAmount = delegators[del.delegateAddress].delegatedAmount.add(amount);
 
-        if (transcoderStatus(del.delegateAddress) == TranscoderStatus.Registered) {
+        // Delete lock
+        delete del.unbondingLocks[_unbondingLockId];
```

Comment:
Why did this statement move?

Reply:
No particular reason, I just though it would be neater to do the stake update after delegator state changes